diktat-maven-plugin/src/test/kotlin/org/cqfn/diktat/plugin/maven/DiktatBaseMojoTest.kt

```diff
@@ -31,15 +49,64 @@ class DiktatBaseMojoTest {
                     
                     <groupId>org.cqfn.diktat</groupId>
                     <artifactId>diktat-test</artifactId>
-                    <version>0.1.6-SNAPSHOT</version>
+                    <version>1.0.0-SNAPSHOT</version>
+                    
+                    <build>
+                        <plugins>
+                            <plugin>
+                                <groupId>org.cqfn.diktat</groupId>
+                                <artifactId>diktat-maven-plugin</artifactId>
+                                <executions>
+                                    <execution>
+                                        <goals>
+                                            <goal>check</goal>
+                                        </goals>
+                                    </execution>
+                                </executions>
+                            </plugin>
+                        </plugins>
+                    </build>
+                </project>
+            """.trimIndent()
+        )
+        val mavenProject = projectBuilder.build(pom, buildingRequest).project
+
+        val diktatCheckMojo = mojoRule.lookupConfiguredMojo(mavenProject, "check") as DiktatCheckMojo
+        Assertions.assertFalse(diktatCheckMojo.debug)
+        Assertions.assertEquals("diktat-analysis.yml", diktatCheckMojo.diktatConfigFile)
+        Assertions.assertIterableEquals(listOf(pom.parentFile.toPath() / "src"), diktatCheckMojo.inputs.map { Path(it) })
```

Comment:
I would prefer a string template here: "${pom.parentFile.toPath()}/src"

Reply:
It's like a killer-feature of kotlin 1.4, an overloaded division operator like in C++ :) And unlike string template, it automatically uses platform-specific separator. But I could use `File` instead.

Reply:
you know never mind, this is fine also